src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import javafx.application.Platform;
+
+import java.io.IOException;
+
+/**
+ * This class runs the Duke program - a task manager for users.
+ * @author Sim Jun Heng
+ * @version CS2103T AY21/22 Sem 2
+ */
+public class Duke {
+
+    private TaskList list = new TaskList();
+    public Duke() throws IOException, DukeException {
+        list = Storage.loadFile();
+    }
+
+    public String getResponse(String input) throws IOException {
```

Comment:
Instead of using so many if-else statements, you could have abstracted out the handling commands part. `Parser` identifies the type of command, and maybe `Ui` could have handled the command.